docs/main/user-guides/upgrade.md

```diff
@@ -8,9 +8,7 @@ keywords:
 description: Upgrade to the latest OpenEBS 2.11.0 version is supported only from v1.0.0 and later.
 ---
 
-Latest stable version of OpenEBS is 3.4.0. Check the release notes [here](https://github.com/openebs/openebs/releases/latest).  
-
-To view the steps for upgrading Mayastor to the latest version, checkout the [Mayastor documentation](https://mayastor.gitbook.io/introduction/additional-information/upgrades).
+Latest stable version of OpenEBS is 3.6.0. Check the release notes [here](https://github.com/openebs/openebs/releases/latest).
```

Comment:
Should we point to the 3.6 specific section here?

Reply:
updating it.